package.json

```diff
@@ -6,7 +6,7 @@
   "module": "src/index.js",
   "scripts": {
     "version": "gulp build --prod && git add .",
-    "test": "snyk test && xo ./src/**/*.js && jest",
+    "test": "xo ./src/**/*.js && jest",
```

Comment:
It seems i had it in https://github.com/WhitestormJS/whitestorm.js/commit/6d0b0fbe75fafd4adc586d0f88a2f711647192a3. Will check